src/runtime.jl

```diff
@@ -234,8 +234,8 @@ function serialize_args(args)
 
         buffer = ray_jll.LocalMemoryBuffer(serialized_arg, serialized_arg_size, true)
         metadata = ray_jll.NullPtr(ray_jll.Buffer)
-        inlined_ids = collect(serializer.object_ids)
-        inlined_refs = ray_jll.GetObjectRefs(worker, StdVector(inlined_ids))
+        inlined_ids = StdVector(collect(serializer.object_ids))::StdVector{ray_jll.ObjectID}
+        inlined_refs = ray_jll.GetObjectRefs(worker, inlined_ids)
         ray_obj = ray_jll.RayObject(buffer, metadata, inlined_refs, false)
```

Comment:
orthogonal to this PR but it seems liek these lines are identical to what's below and fiddly enough that they'd benefit from refactoring

Reply:
I've left these lines duplicated for now as the Python code has minor differences. If we adopt those differences in our Julia code they will differ.